### ISSUE TITLE
Missing wifi drivers on macbook pro in openfyde but present on fyde OS 15.1

### ISSUE THREAD

**Issue by enkiros** [openFyde/overlay-amd64-openfyde#6]
FydeOS works perfectly on a 2011 macbook pro (I believe it has an intel sandy bridge cpu) plus wifi and Bluetooth work as intended. 
The wifi drivers are completely missing on openfyde. I tried r102, 102r1, and 102r2 just in case.
Are they not included because they're proprietary drivers?
Is there a way I can include them and then build openfyde?

**Comment by Damenly** [comment]
Right, proprietary drivers. You can writes a package ebuild like What Gentoo does.
